crates/egui/src/widgets/text_edit/builder.rs

```diff
@@ -67,6 +67,8 @@ pub struct TextEdit<'t> {
     desired_height_rows: usize,
     lock_focus: bool,
     cursor_at_end: bool,
+    align: Align,
+    use_content_width: bool,
```

Comment:
Can we find a better name for `use_content_width`, like `follow_text_width`? Or maybe the opposite makes more sense `clip_text: bool` ?